graphsdk/src/androidTest/java/com/microsoft/graph/http/BaseCollectionPageTests.java

```diff
@@ -0,0 +1,88 @@
+package com.microsoft.graph.http;
+
+import android.test.AndroidTestCase;
+
+import com.google.gson.JsonObject;
+import com.microsoft.graph.core.IBaseClient;
+import com.microsoft.graph.core.MockBaseClient;
+import com.microsoft.graph.serializer.ISerializer;
+import com.microsoft.graph.serializer.MockSerializer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Test cases for {@see BaseCollectionPage}
+ */
+public class BaseCollectionPageTests extends AndroidTestCase {
+
+    private IRequestBuilder mRequestBuilder;
+    private static ArrayList<String> list;
+    private BaseCollectionPage baseCollectionPage;
+    private String requestUrl = "https://graph.microsoft.com/v1.0/me";
+
+    @Override
+    @SuppressWarnings("unchecked")
```

Comment:
Please do this whenever possible in all tests cases

---
In reply to: [65748765](https://github.com/microsoftgraph/msgraph-sdk-android/pull/17#discussion_r65748765) [](ancestors = 65748765)